packages/bsky/src/services/indexing/plugins/post.ts

```diff
@@ -170,6 +233,14 @@ const eventsForInsert = (obj: IndexedPost) => {
       )
     }
   }
+
+  if (obj.descendents) {
+    // May generate notifications for out-of-order indexing of replies
```

Comment:
yup yup that makes sense. the timing on these if they're out of order should be close anyway 👌